### ISSUE TITLE
Mismatch database when update cve-search

### ISSUE THREAD

**Issue by longnv519** [eclipse/sw360#401]
In wiki:
>  After a CVE-search run has been finished, you can see the number of vulnerabilities associated with your project in the Projects Portlet. To that end, open the Projects Portlet and click on your project:
In the Vulnerabilities tab on the left hand side, you see the number of vulnerabilites that have been found for the releases that are directly linked to your project

- After i finished to run CVE-search update, i just call:
  ```
  final CveSearchService.Iface cveSearchClient = thriftClients.makeCvesearchClient();
  RequestStatus requestStatus = cveSearchClient.update();
  ```
  , many object vulnerability has been added in sw360db, but i still can not see any number on the Vulnerabilities tab?
- When add new vulnerability to database, sw360 call wrong database name `sw360db`, it's must be `sw360vm`

**Comment by longnvtoshiba** [comment]
My environment need to setting proxy, so i just use http://cve.circl.lu instead of https://cve.circl.lu. But i think it's not problem

**Comment by longnvtoshiba** [comment]
I saw in Vulnerabilities tab, CVE search is using sw360vm as database. But why when we update CVE-search, nothing is written to sw360vm?

**Comment by mcjaeger** [comment]
Regarding the last point it is that there should be a setting ion the couchdb.properties, such as:

https://github.com/eclipse/sw360/blob/master/build-configuration/resources/couchdb.properties

which could be placed inside the `*.war` file of the deployed backend service or it will be overridden by a manually placed `couchdb.properties`  in `/etc/sw360`

**Comment by mcjaeger** [comment]
(from dev telco today)

the scheduling ofthe CVE sync is done with the scheduling service, which could be called via thrift or just in the UI:

![image](https://user-images.githubusercontent.com/3418977/49154130-f2e9b500-f317-11e8-9d96-6c81c78cefa0.png)

**Comment by longnvtoshiba** [comment]
Here you go, it's written in:

[](https://github.com/eclipse/sw360/blob/master/backend/src/src-cvesearch/src/main/java/org/eclipse/sw360/cvesearch/datasink/VulnerabilityConnector.java)

At line 46, init `VulnerabilityDatabaseHandler` is using `DatabaseSettings.COUCH_DB_DATABASE` variable as target database then all of vulnerabilities matched will be saved to `sw360db`.

**Comment by maxhbr** [comment]
The vulnerabilies are saved in the `DatabaseSettings.COUCH_DB_VM` and the relation between releases in sw360 and vulnerabilities is saved in `DatabaseSettings.COUCH_DB_DATABASE`. This might explain the confusion.

**Comment by imaykay** [comment]
I can confirm this bug! Currently 
* the scheduler (CveSearchHandler) uses the VulnerabilityConnector which creates a VulnerabilityDatabaseHandler with the standard sw360db database

while
* the thrift service (VulnerabilityHandler) creates a VulnerabilityDatabaseHandler with the sw360vm database.

This means that all vuls and their relations created by the scheduler are stored in the standard db whereas the thrift services try to find both in the vuls db.

If I understood @maxhbr correctly the desired behavior would be that the vuls itself should go into the separate vuls db while the relations to releases should go into the standard db. I would try to create a PR for this.

**Comment by imaykay** [comment]
I changed my mind during implementation. I only did a quick fix in the way that both paths are using the vm db which was currently used by the thrift services. This way we do not break productive instances. If I implemented the final solution where we would split the information in both dbs, we would need some migration for running instances.

We should create another issue if we really want to achieve this distinction - in this issue it was important to make the scheduler useable again. So, do we really want the split of the information, meaning vulnerabilities in their own db and release relations in the standard db? /cc @mcjaeger @maxhbr
